cron/client.py

```
@@ -106,10 +111,15 @@ def reboot_if_uptime_exceeded(reboot_after_s):
 
     if uptime_seconds > reboot_after_s:
         print(f"Uptime {uptime_seconds:.0f}s exceeds reboot_after_seconds {reboot_after_s}s. Rebooting...")
-        set_status('reboot')
-        subprocess.check_output(['sync'], encoding='UTF-8', errors='replace')
-        subprocess.check_output(['/usr/bin/sudo', '/usr/bin/systemctl', 'reboot'], encoding='UTF-8', errors='replace')
-        time.sleep(86400) # reboot request might not be handled directly, thus we wait until this process gets killed.
+        systemctl_change_state('reboot')
+
+def systemctl_change_state(state, wait=False):
+    subprocess.check_output(['sync'], encoding='UTF-8', errors='replace')
```

Ruff (0.16.2)

[error] Starting a process with a partial executable path

(S607)